src/node-uriparser.cc

```diff
@@ -260,6 +229,10 @@ void init (v8::Handle<v8::Object> target){
     NODE_DEFINE_CONSTANT(target, kFragment);
     NODE_DEFINE_CONSTANT(target, kPath);
     NODE_DEFINE_CONSTANT(target, kAll);
+
+    NODE_DEFINE_CONSTANT(target, eNgxParser);
```

Comment:
Before there were no constants of different type, since now there are so we should put them in new object.